src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.exception.DataProcessingException;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final DtoResponseMapper<UserResponseDto, User> userDtoResponseMapper;
```

Comment:
better to name it just as an actual implementation class is named, same for other dependencies
```suggestion
    private final DtoResponseMapper<UserResponseDto, User> userResponseMapper;
```